### ISSUE TITLE
Getting "Failed to complete tunnel connection" error on clicking ngrok url.

### ISSUE THREAD

**Issue by rohitar** [abhishekkrthakur/colabcode#38]
This was working fine until yesterday. The error message is attached. I'm running colabcode on Google Colab. Error seems to suggest that it is trying to establish connection with my localhost not sure why as I'm running it on colab.
![colab-code-error](https://user-images.githubusercontent.com/16827390/97114104-87cbd780-1714-11eb-80dd-2ca78aec39f2.png)

**Comment by PiotrekGa** [comment]
I have the same problem too. It worked well this morning, but then it stopped. I've updated my Chrome browser (on MacOS).

**Comment by GJvignesh** [comment]
Same problem

**Comment by GJvignesh** [comment]
Try Port 80, its working fine.

from colabcode import ColabCode
ColabCode(port= 80, mount_drive=True)

**Comment by abhishekkrthakur** [comment]
This has been fixed in version 0.0.9.

`pip install -U colabcode`

Please let me know if its fixed and I will close the issue.

**Comment by GJvignesh** [comment]
Thanks @abhishekkrthakur . Issue is fixed !

Commands:
`! pip install colabcode==0.0.9`
or
`pip install -U colabcode`

```
from colabcode import ColabCode
ColabCode(port= 10000, mount_drive=True)
```
